pkg/api/login.go

```diff
@@ -30,8 +30,9 @@ import (
 )
 
 const (
-	viewIndex            = "index"
-	loginErrorCookieName = "login_error"
+	viewIndex               = "index"
+	loginErrorCookieName    = "login_error"
+	postLogoutRedirectParam = "?post_logout_redirect_uri="
```

Comment:
If we use this way, then `post_logout_redirect_uri` should be first QueryParameter. It might be difficult to debug.

Possible QueryParameters can be found here https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout

There are 5(6) Possible QueryParamater anything can come in front

Reply:
Docs also needs to be updated with steps to configure RP initiated logout.

Reply:
i can search for post_logout_redirect_uri=. Would that be fine?

Reply:
It can work, but If I send my query as `?client_post_logout_redirect_uri=%2fhome&client_id=app` it will pass the test. But It is a very rare case.

If there is a regex to queryparam key this then It will be best option.